conference.js

```diff
@@ -468,14 +468,18 @@ export default {
             audioOnlyError,
             screenSharingError,
             videoOnlyError;
-        const initialDevices = [];
-        let requestedAudio = false;
+        const initialDevices = [ 'audio' ];
+        const requestedAudio = true;
         let requestedVideo = false;
 
-        if (!options.startWithAudioMuted) {
-            initialDevices.push('audio');
-            requestedAudio = true;
+        // Always get a handle on the audio input device so that we have statistics even if the user joins the
+        // conference muted. Previous implementation would only acquire the handle when the user first unmuted,
+        // which would results in statistics ( such as "No audio input" or "Are you trying to speak?") being available
+        // only after that point.
+        if (options.startWithAudioMuted) {
```

Comment:
There is one more aspect of this that I'm worried about. If you join for first time with audio muted the browser prompt will ask you to give audio permissions too. Have you confirm with @emcho or @yanas  that this is OK?

Reply:
Yes, I confirmed this with @emcho . The consequence of not having this is that detection won't work until you unmute a first time, so no "talk while muted" notification and no "no audio signal".

Reply:
@hristoterezov @andrei-gavrilescu I'm a little bit confused, In the last call @emcho was talking about a view only mode where you can not hear anything and in that case you won't have to accept permissions, is this something new which renders this convo moot? https://youtu.be/QERTcdf1RIk?t=2696 

Or is this a different config?

Reply:
This conversation is from 2 years ago. What you ask for seems to be `startSilent`

Reply:
For future reference, this worked for me:
`disableInitialGUM: true`

Got it from here:
https://community.jitsi.org/t/dont-ask-for-mic-permission-in-startwithaudiomuted/48132/10?u=chagai_friedlander